### PR TITLE
MAUI-4346-UG content for .NET MAUI Charts radial bar series

### PR DESCRIPTION
**Pull request by NanthiniMahalingam**
### Feature description ###

To prepare user guide document for radial bar chart support

### Solution description ###

- I Created the UG Document in md Format named as RadialBarBar.md
- I attached a folder contains the output images for Radial Bar Series
- I attached the images to the md file
- I added the Radial Bar in the html List file

### Areas affected and ensured ###

**None**

### API Changes ###

https://syncfusion.atlassian.net/browse/MAUI-3336 

### Behavioral Changes ###

**None**

### Test cases ###

[**Test plan link**
](https://tcms.syncfusion.com/plan/6149/)

### Testbed sample location ###

I prepared test bed sample.
Location : [**Sample.**](https://github.com/NanthiniMahalingam/WorkArea/tree/main/Vol3-2022/RadialBarChart/SampleFile)

### Does it have any known issues?

**None**

### Does it have memory leak?

**None**

### MR CheckList ###

- [x] Does it follow the design [guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/)? It is mandatory that, we should not move the changes without reading this. 
- [ ] Did UI automation passed without errors? If it is customer issue, make sure it is included in the IR automation. **No**
- [ ] Properly working in Xamarin.Forms [previewer](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/xaml/xaml-previewer?tabs=vswin). **No**
- [x] Ensured in iOS, Android, UWP and macOS (if supported). **Yes**
- [x] Have you added API comments if it is new API? **Yes**
- [ ] Does it need localization? If so, update resx file in [GitHub](https://github.com/syncfusion/xamarin-localized-texts).
- [ ] Whether we have added new APIs? If so, whether it is as per the [guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/)? **No**
- [ ] If changes are made in the Xamarin.Forms sample browser project file, whether it is included in the props file too? If not, your sample level changes will not reflect in the application published in the hockey app. **No**
- [ ] Have you ensured the changes in Android API 19 and iOS 9? **No**

### CODE REVIEW

File: MAUI/Circular-Charts/RadialBarChart.md [hunk 1]
```diff
@@ -0,0 +1,433 @@
...
+        series.YBindingPath = "SalesRate";
+
+        chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Radial bar chart type in MAUI Chart](Chart-Types_images/maui_radialbar_chart.png)
+
```
Comment (MuneeshKumarG) [inline MAUI/Circular-Charts/RadialBarChart.md:58]: Image text is not proper

File: MAUI/Circular-Charts/RadialBarChart.md [hunk 2]
```diff
@@ -0,0 +1,433 @@
...
+## Changing the radial bar size
+
+You can use the [Radius]() property to change the radial bar chart size. The default value of the radius is [0.8]().
+
```
Comment (MuneeshKumarG) [inline MAUI/Circular-Charts/RadialBarChart.md:62]: Add link for Radius. 
Default value 0.8 syntax is wrong.

File: MAUI/Circular-Charts/RadialBarChart.md [hunk 3]
```diff
@@ -0,0 +1,433 @@
...
+        series.Radius = 0.5;
+
+        chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Radial bar chart radius size in MAUI Chart](Chart-Types_images/maui_radius.png)
+
```
Comment (MuneeshKumarG) [inline MAUI/Circular-Charts/RadialBarChart.md:92]: Change  to .NET MAUI Radial bar chart radius customization.

File: MAUI/Circular-Charts/RadialBarChart.md [hunk 4]
```diff
@@ -0,0 +1,433 @@
...
+        series.InnerRadius = 0.2;
+
+        chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Radial bar chart inner radius customization in MAUI Chart](Chart-Types_images/maui_inner_radius.png)
+
```
Comment (MuneeshKumarG) [inline MAUI/Circular-Charts/RadialBarChart.md:126]: Change to ".NET MAUI Radial bar chart inner radius customization"

File: MAUI/Circular-Charts/RadialBarChart.md [hunk 5]
```diff
@@ -0,0 +1,433 @@
...
+        series.CapStyle = CapStyle.BothFlat;
+
+        chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Radial bar chart both flat cap style in MAUI Chart](Chart-Types_images/maui_radialbar_chart.png)
+
```
Comment (MuneeshKumarG) [inline MAUI/Circular-Charts/RadialBarChart.md:172]: .NET MAUI Radial bar chart both flat cap style.

File: MAUI/Circular-Charts/RadialBarChart.md [hunk 6]
```diff
@@ -0,0 +1,433 @@
...
+        series.CapStyle = CapStyle.BothCurve;
+
+        chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Radial bar chart both curve cap style in MAUI Chart](Chart-Types_images/maui_bothcurve.png)
+
```
Comment (MuneeshKumarG) [inline MAUI/Circular-Charts/RadialBarChart.md:204]: .NET MAUI Radial bar chart both curve cap style.

File: MAUI/Circular-Charts/RadialBarChart.md [hunk 7]
```diff
@@ -0,0 +1,433 @@
...
+        series.CapStyle = CapStyle.StartCurve;
+
+        chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Radial bar chart start curve cap style in MAUI Chart](Chart-Types_images/maui_startcurve.png)
+
```
Comment (MuneeshKumarG) [inline MAUI/Circular-Charts/RadialBarChart.md:236]: .NET MAUI Radial bar chart start curve cap style

File: MAUI/Circular-Charts/RadialBarChart.md [hunk 8]
```diff
@@ -0,0 +1,433 @@
...
+ * [BothFlat]() - Indicates that a flat shape should appear at the start and end positions of the segment.
+
+ * [BothCurve]() - Indicates that a curve shape should appear at the start and end positions of the segment. 
+
+ * [StartCurve]() - Indicates that a curve shape should appear at the start position of the segment.
+
+ * [EndCurve]() - Indicates that a curve shape should appear at the end position of the segment.
+
+### BothFlat
```
Comment (MuneeshKumarG) [inline MAUI/Circular-Charts/RadialBarChart.md:141]: Remove all sub headings and use single code snippet and image for both curve alone.

File: MAUI/Circular-Charts/RadialBarChart.md [hunk 9]
```diff
@@ -0,0 +1,433 @@
...
+        series.GapRatio = 0.4;
+
+        chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Radial bar chart segment spacing in MAUI Chart](Chart-Types_images/maui_gapratio.png)
+
```
Comment (MuneeshKumarG) [inline MAUI/Circular-Charts/RadialBarChart.md:304]: .NET MAUI Radial bar chart segment spacing customization.

File: MAUI/Circular-Charts/RadialBarChart.md [hunk 10]
```diff
@@ -0,0 +1,433 @@
...
+You can use the following properties to customize the appearance of the circular bar track.
+
+ * [TrackStroke]() - To customize the circular bar border color.
+
+  * [TrackStrokeWidth]() - To customize the border width of the circular bar.
+
+  * [TrackFill]() - To customize the circular bar area which behind the radial bar segments.
+
+### TrackStrokeWidth
+
```
Comment (MuneeshKumarG) [inline MAUI/Circular-Charts/RadialBarChart.md:316]: Remove this section.

File: MAUI/Circular-Charts/RadialBarChart.md [hunk 11]
```diff
@@ -0,0 +1,433 @@
...
+{% endhighlight %}
+
+{% endtabs %}
+
+
+![Radial bar chart track fill in MAUI Chart](Chart-Types_images/maui_trackfill.png)
+
+## CenterView
+
+The view placed in the center of the radial bar chart is useful for sharing additional information about the radial bar chart. Any view can be added to the center of the radial bar chart using the [CenterView]() property of [RadialBarSeries](). The binding context of the [CenterView]() will be the respective radial bar series.
```
Comment (MuneeshKumarG) [inline MAUI/Circular-Charts/RadialBarChart.md:385]: Swap the first and second sentence.

File: MAUI/Circular-Charts/RadialBarChart.md [hunk 12]
```diff
@@ -0,0 +1,433 @@
...
+        layout.Children.Add(image);       
+
+        series.CenterView = image;
+        chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Radial bar chart center view in MAUI Chart](Chart-Types_images/maui_radialbarchart_centerview.png)
```
Comment (MuneeshKumarG) [inline MAUI/Circular-Charts/RadialBarChart.md:433]: .NET MAUI Radial bar chart center view

File: MAUI/Circular-Charts/RadialBarChart.md (hunk 1 above)
Comment (NanthiniMahalingam) [reply MAUI/Circular-Charts/RadialBarChart.md:57]: I have changed the image alt text.

File: MAUI/Circular-Charts/RadialBarChart.md (hunk 2 above)
Comment (NanthiniMahalingam) [reply MAUI/Circular-Charts/RadialBarChart.md:61]: I have added the radius property link and added proper syntax.

File: MAUI/Circular-Charts/RadialBarChart.md (hunk 3 above)
Comment (NanthiniMahalingam) [reply MAUI/Circular-Charts/RadialBarChart.md:91]: I have changed to image alt text.

File: MAUI/Circular-Charts/RadialBarChart.md (hunk 4 above)
Comment (NanthiniMahalingam) [reply MAUI/Circular-Charts/RadialBarChart.md:125]: I have changed the inner radius image alt text.

File: MAUI/Circular-Charts/RadialBarChart.md (hunk 9 above)
Comment (NanthiniMahalingam) [reply MAUI/Circular-Charts/RadialBarChart.md:209]: I have changed the gap ratio image alt text.

File: MAUI/Circular-Charts/RadialBarChart.md (hunk 12 above)
Comment (NanthiniMahalingam) [reply MAUI/Circular-Charts/RadialBarChart.md]: I have changed the  center view image alt text

File: MAUI/Circular-Charts/RadialBarChart.md (hunk 11 above)
Comment (NanthiniMahalingam) [reply MAUI/Circular-Charts/RadialBarChart.md]: I changed the sentence arrangements.

File: MAUI/Circular-Charts/RadialBarChart.md (hunk 10 above)
Comment (NanthiniMahalingam) [reply MAUI/Circular-Charts/RadialBarChart.md]: I have removed the track customisation sub headings

File: MAUI/Circular-Charts/RadialBarChart.md (hunk 8 above)
Comment (NanthiniMahalingam) [reply MAUI/Circular-Charts/RadialBarChart.md]: I have removed all the sub heading in cap style customization  section

File: MAUI/Circular-Charts/RadialBarChart.md (hunk 6 above)
Comment (NanthiniMahalingam) [reply MAUI/Circular-Charts/RadialBarChart.md]: I have changed the cap style image alt text.

**Review by MuneeshKumarG — APPROVED** [review]
This is fine

**Pull request merged**